### PR TITLE
Framework enablement

### PR DESCRIPTION
**Pull request by tsudd**
The project was updated using React. New version is [available now](https://dreva-tsudd.web.app/).
You can use credentials provided down below to check data, statistics for generic user:
email: hristenko2002@gmail.com
password: 138566sva

### CODE REVIEW

File: dreva/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (katetushkan) [inline dreva/.gitignore:1]: Screenshot 2022-05-16 at 15 27 41
I think u passed wrong value)) swap it

Comment (katetushkan) [reply dreva/.gitignore:1]: Screenshot 2022-05-16 at 15 29 31
Not able to login(

**Review by katetushkan — APPROVED** [review]

File: dreva/.gitignore (hunk 1 above)
Comment (tsudd) [reply dreva/.gitignore:1]: I'm sorry(
It should be hristenko2002@mail.ru
Same password.

**Review by katetushkan — APPROVED** [review]
<kbd>10</kbd> is urs.

**Pull request merged**